setup.cfg

```diff
@@ -0,0 +1,22 @@
+[coverage:run]
```

Comment:
I would remove `setup.cfg` entirely. Seems weird to me to have two files for configuration. It seems like `setup.cfg` is an artifact of the old setuptools build system.